homeassistant/components/plugwise/config_flow.py

```diff
@@ -40,9 +40,19 @@ async def validate_input(hass: core.HomeAssistant, data):
     Data has the keys from _base_schema() with values provided by the user.
     """
     websession = async_get_clientsession(hass, verify_ssl=False)
+
+    port = 80
+    host = data[CONF_HOST]
+    password = data[CONF_PASSWORD]
+
+    if ":" in host:
```

Comment:
The port should not be a part of the `host` key, this should be a separate entry like all other integrations.

Reply:
It will still have 80 as the default? so they will just keep it as is.
If a port is needed to be configured, it should have dedicated entry.

Reply:
Re-adjusting to #38276 (by @jeroen84) for the most part, still needs additional local testing and docs-PR adjustment.